Kudu.Services/Diagnostics/ProfileManager.cs

```diff
@@ -18,16 +18,24 @@ internal static class ProfileManager
     {
         private const int ProcessExitTimeoutInSeconds = 180;
 
+        private const string ServiceProfilerAgentGuid = "F5091AA9-80DC-49FF-A7CF-BD1103FE149D";
```

Comment:
Let's call this ```UserModeCustomProviderAgentGuid```. (While it is bundled in the Service Profiler DLL, that's just for convenience.)
Once you've done that rename, then it frees up the name ```ServiceProfilerAgentGuid``` for what is currently ```ServiceProfilerProviderGuid``` (1C92B...)